src/payment/payment.ts

```diff
@@ -32,6 +32,8 @@ export interface VaultedInstrument {
     instrumentId: string;
     ccCvv?: string;
     ccNumber?: string;
+    iin: string;
+    threeDSecure?: ThreeDSecure | ThreeDSecureToken;
```

Comment:
Changes related to `3DS` shouldn't be in this PR since they were already merged

Reply:
Done 👍